scripts/secnetperf.ps1

```diff
@@ -257,7 +257,7 @@ $allTests = [System.Collections.Specialized.OrderedDictionary]::new()
 $allTests["tput-up"] = "-exec:maxtput -up:12s -ptput:1"
 $allTests["tput-down"] = "-exec:maxtput -down:12s -ptput:1"
 $allTests["hps-conns-100"] = "-exec:maxtput -rconn:1 -share:1 -conns:100 -run:12s -prate:1"
-$allTests["rps-up-512-down-4000"] = "-exec:lowlat -rstream:1 -up:512 -down:4000 -run:20s -plat:1"
+$allTests["rps-up-512-down-4000"] = "-exec:lowlat -rstream:1 -up:512 -down:4000 -run:20s -plat:1 -abortOnFailure"
```

Comment:
Do we really need to abort on failure by default in netperf?

Reply:
I have this as a temporary change to collect the dumps and fiddle with GDB. Will revert this.